### ISSUE TITLE
Erroneous NS1002 errors for Arg.Any calls

### ISSUE THREAD

**Issue by ksstott** [nsubstitute/NSubstitute.Analyzers#91]
I've got some code like this:
```
var myMock = Substitute.For<IDependancy>();
myMock.When(a => a.Foo(Arg.Any<SomeClass>())).Throw(new Exception());
```

Since updating to version 1.0.8 we're getting the following error on the `.When` line.

`error NS1002: Member Any can not be intercepted. Only interface members and virtual, overriding, and abstract members can be intercepted.`

**Comment by tpodolak** [comment]
@ksstott thanks for reporting, looks like this is regression after the last changes which were aiming to fix https://github.com/nsubstitute/NSubstitute.Analyzers/issues/89 Will try to look at this today. If this is not a problem please downgrade for now to 1.0.7

**Comment by tpodolak** [comment]
@ksstott PR waiting for review. If everything will be ok, I will release it either tonight or tomorrow morning

**Comment by tpodolak** [comment]
Fix released in [1.0.9](https://www.nuget.org/packages/NSubstitute.Analyzers.CSharp/1.0.9)
